packages/app/src/specs/generators/FileChooser.cy.tsx

```diff
@@ -237,10 +237,10 @@ describe('<FileChooser />', () => {
     .should('have.value', newExtension)
 
     // debounce should cause this to hit
-    .get('@onUpdateExtensionSpy').should('not.have.been.calledWith', newExtension)
+    cy.get('@onUpdateExtensionSpy').should('not.have.been.calledWith', newExtension)
```

Comment:
Same as above, would have thought `.type()` broken the change so this change wouldn't have been needed.